random-beans/src/test/java/io/github/benas/randombeans/CollectionPopulationTest.java

```diff
@@ -231,11 +229,6 @@ public void compositeCollectionTypesShouldBeEmpty() {
     }
 
     @Test(expected = ObjectGenerationException.class)
-    public void synchronousQueueTypeMustBeRejected() {
```

Comment:
This is a functional test about unsupported `SynchronousQueue` type. Please keep it